python/lsst/summit/extras/logUtils.py

```diff
@@ -58,15 +59,16 @@ class property ``SPECIAL_ZOO_CASES`` to declare a new type of error animal.
         # butler.datastores is verbose by default and not interesting to most
         'lsst.daf.butler.datastores',
     ]
-    SPECIAL_ZOO_CASES = ['with gufunc signature (n?,k),(k,m?)->(n?,m?)',
-                         ]
+    SPECIAL_ZOO_CASES = [
+        'with gufunc signature (n?,k),(k,m?)->(n?,m?)',
+    ]
```

Comment:
PS - I think my comment may have been lost in the noise, but you can wrap this onto one line if you wish (the trailing comma led me to put it as above, but one line works just fine).